device-types/Fortinet/FG-100D.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100D
+slug: fg-100d
+part_number: FG-100D
+u_height: 1.0
+is_full_depth: false
```

Comment:
Please add `airflow` information if you have it

Reply:
5c7212ee71eabf6505cfa0a7ff08bce8530753a1